desc/coils.py

```diff
@@ -950,24 +1005,39 @@ def from_symmetry(cls, coils, NFP, sym=False):
         Parameters
         ----------
         coils : Coil, CoilGroup, Coilset
-            base coil or collection of coils to repeat
-        NFP : int
-            number of field periods
-        sym : bool
-            whether coils should be stellarator symmetric
+            Coil or collection of coils in one field period or half field period.
+        NFP : int (optional)
```

Comment:
Should clarify that this explicitly creates extra coils, not "virtual" coils like the regular init method